src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -44,6 +46,11 @@ private Constants() {
     public static final Integer DELAY_BETWEEN_BUFFER_TASKS = 5000;
     public static final Integer DELAY_BETWEEN_BUFFER_TASKS_IN_SECONDS = 5;
 
+    //Database queries and strings
+    public static final String DATABASE_PATH = Paths.get(CODESYNC_ROOT, "codesyncdb.db").toString();
+    public static final String CONNECTION_STRING = "jdbc:sqlite:" + DATABASE_PATH;
+    public static final String CREATE_USER_TABLE_QUERY = "CREATE TABLE IF NOT EXISTS user (EMAIL TEXT PRIMARY KEY, ACCESS_TOKEN TEXT, SECRET_KEY TEXT, ACCESS_KEY TEXT, IS_ACTIVE INT)";
```

Comment:
I think you have moved this to separate file, maybe need to remove from here @syedgulahmed